forge-ai/src/main/java/forge/ai/SpecialCardAi.java

```diff
@@ -78,16 +78,17 @@ public class SpecialCardAi {
 
     // Arena and Magus of the Arena
     public static class Arena {
-        public static boolean consider(final Player ai, final SpellAbility sa) {
+        public static AiAbilityDecision consider(final Player ai, final SpellAbility sa) {
```

Comment:
it seems like some of the `consider` functions here have been arbitrarily skipped

maybe there's some kind of reasoning but then it should probably be documented there at least